src/Descriptors/Server.php

```diff
@@ -3,23 +3,36 @@
 namespace LaravelJsonApi\OpenApiSpec\Descriptors;
 
 use GoldSpecDigital\ObjectOrientedOAS\Objects;
+use Illuminate\Support\Arr;
 use LaravelJsonApi\OpenApiSpec\Descriptors\Descriptor as BaseDescriptor;
 
 class Server extends BaseDescriptor
 {
     /**
      * @return \GoldSpecDigital\ObjectOrientedOAS\Objects\Info
-     *
-     * @todo Add contact
-     * @todo Add TOS
-     * @todo Add License
      */
     public function info(): Objects\Info
     {
         return Objects\Info::create()
-          ->title(config("openapi.servers.{$this->generator->key()}.info.title"))
-          ->description(config("openapi.servers.{$this->generator->key()}.info.description"))
-          ->version(config("openapi.servers.{$this->generator->key()}.info.version"));
+            ->title(config("openapi.servers.{$this->generator->key()}.info.title"))
+            ->description(config("openapi.servers.{$this->generator->key()}.info.description"))
+            ->version(config("openapi.servers.{$this->generator->key()}.info.version"))
+            ->termsOfService(config("openapi.servers.{$this->generator->key()}.info.termsOfService"))
+            ->license(
+                Arr::has(config("openapi.servers.{$this->generator->key()}.info"), 'license')
+                    ? Objects\License::create()
+                    ->name(config("openapi.servers.{$this->generator->key()}.info.license.name"))
```

Comment:
Could you add an indentation to the methods of the License object to improve readability?